ext/json/Makefile.frag

```diff
@@ -2,4 +2,4 @@ $(srcdir)/json_scanner.c: $(srcdir)/json_scanner.re
 	$(RE2C) -t $(srcdir)/php_json_scanner_defs.h --no-generation-date -bci -o $@ $(srcdir)/json_scanner.re
 
 $(srcdir)/json_parser.tab.c: $(srcdir)/json_parser.y
-	$(YACC) --defines -l $(srcdir)/json_parser.y -o $@
+	$(BISON) --defines -l $(srcdir)/json_parser.y -o $@
```

Comment:
I see that main Makefile defines YACC, but I don't see anywhere it defines BISON. Where that comes from?

Reply:
yes you are right, BISON is not defined. The line is "broken" with this patch. It does compile and seems to work fine when compiled as module or into PHP, so I wonder what this line is for. But something is not right. I am investigating this further ...